src/Dhall/Import.hs

```diff
@@ -470,17 +460,17 @@ removeAtFromURL url
     | Text.isSuffixOf "/"  url = Text.dropEnd 1 url
     | otherwise                =                url
 
-removeAtFromFile :: FilePath -> FilePath
-removeAtFromFile file =
-    if Filesystem.filename file == "@"
-    then Filesystem.parent file
-    else file
+removeAtFromFilename :: FilePath -> FilePath
+removeAtFromFilename fp =
+    if FilePath.takeFileName fp == "@"
+    then FilePath.takeDirectory fp
+    else fp
 
 -- | Remove all @.@'s and @..@'s in the path
 clean :: FilePath -> FilePath
-clean = strip . Filesystem.collapse
+clean = strip . FilePath.normalise
   where
-    strip p = case Filesystem.stripPrefix "." p of
+    strip p = case List.stripPrefix "." p of
```

Comment:
I haven't already but I guess I also need to change "." for "./" here ?

Reply:
Do we still need to call `strip` ? (tests are passing either way)

Reply:
You're right that we don't need `stripPrefix` here because `System.FilePath.normalise` removes any preceding `./`s (unlike `Filesystem.Path.collapse`)